simhash/simhash.pyx

```diff
@@ -1,9 +1,13 @@
+import hashlib
+import struct
+
 from simhash cimport compute as c_compute
 from simhash cimport find_all as c_find_all
 
-def unsigned_hash(obj):
-    '''The built-in hash suitable for use as a hash_t.'''
-    return hash(obj) & 0xFFFFFFFFFFFFFFFF
+
+def unsigned_hash(bytes obj):
+    '''Returns a hash suitable for use as a hash_t.'''
+    return struct.unpack('>Q', hashlib.md5(obj).digest()[0:8])[0] & 0xFFFFFFFFFFFFFFFF
```

Comment:
The `struct.pack` and `.unpack` methods are pretty cryptic. How about a comment?